SU2_CFD/src/output/COutput.cpp

```diff
@@ -2181,6 +2165,130 @@ void COutput::SetCommonHistoryFields() {
 
 }
 
+void COutput::SetCustomOutputs(const CConfig* config) {
+
+  const auto& inputString = config->GetCustomOutputs();
+  if (inputString.empty()) return;
+
+  /*--- Split the different functions. ---*/
+
+  auto DebugPrint = [](const std::string& str) {
+#ifndef NDEBUG
+    std::cout << str << std::endl;
+#endif
```

Comment:
Keeping additional debugging code behind preprocessor directives seems like an concept Su2 could adopt... but if more people use NDEBUG, that might quickly bloat screen output. (so maybe some additional info needs to be printed ahead? idk just leave it like that, i am more thinking to myself)